### ISSUE TITLE
Control node show/hide from ModelGraphics

### ISSUE THREAD

**Issue by OmarShehata** [AnalyticalGraphicsInc/cesium#7761]
Right now at the entity layer, you can control each node's scale, rotation and translation using the [NodeTransformationProperty](https://cesiumjs.org/Cesium/Build/Documentation/NodeTransformationProperty.html). This allows you to do cool things like make the [wheels on a vehicle rotate programmatically to match its speed](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/?src=Time%20Dynamic%20Wheels.html).

However, there's no way to programmatically toggle `show` for the nodes. To do this, you'll have to rewrite your application to use primitives (and thus use all the time dynamic properties you might be relying on). Using the primitive it's simply `ModelPrimitive.getNode('nodeName').show = false`. 

It would be easy to extend `NodeTransformationProperty` to add this, but the name implies that it's just a transformation property. So perhaps a new property on ModelGraphics? 

This has come up in a project I was recently working on, and just came up [on the forum here](https://groups.google.com/d/msg/cesium-dev/9eXPOxAvRV0/P4yNQm5xCQAJ).

**Comment by shunter** [comment]
I think if you treat the term as a general term rather than a graphics term, "transformation" works to mean any change applied to a node.  It seems sensible to me that NodeTransformation could toggle visibility, by adding more properties to the existing object, making NodeTransformation the way to represent any modification that is specific to a single node rather than the whole model.

**Comment by OmarShehata** [comment]
I think that would be nicer from an API perspective, to have one PropertyBag that lets you control everything about a node. Then it can all easily be time dynamic too. It should be pretty easy to add this.

**Comment by Wyckliffe** [comment]
Thank you, Omar, for opening this issue. 

My problem is rather simple because the parts I would like to toggle on and off are actually primitive models that is: 3 pyramids and 2 cones.  When I open the gltf file on text editor though I see 182 nodes do you know how I can separate them and find the 5 I am interested in?

**Comment by OmarShehata** [comment]
@Wyckliffe yes! I've run into exactly this sort of thing before. You'll have to re-organize your nodes in something like Blender, by selecting all parts you consider to be one whole and grouping them under one parent. This parent can then have a node name that you can toggle. See https://docs.blender.org/manual/en/latest/editors/3dview/object/properties/relations/parents.html

**Comment by Wyckliffe** [comment]
Thank you very much I will get on this and keep you updated.
